Sming/Makefile

```diff
@@ -192,13 +192,27 @@ ifeq ($(ENABLE_CUSTOM_HEAP), 1)
 	CUSTOM_TARGETS += $(USER_LIBDIR)/libmainmm.a
 endif
 
+LWIP_FLAVOUR ?= esp-open-lwip
+#LWIP_FLAVOUR ?= lwip2
+
 # => Open Source LWIP
 LIBLWIP = lwip
 ENABLE_CUSTOM_LWIP ?= 1
 ENABLE_ESPCONN ?= 0
 ifeq ($(ENABLE_CUSTOM_LWIP), 1)
-	THIRD_PARTY_DATA += third-party/esp-open-lwip/Makefile.open
-	EXTRA_INCDIR += third-party/esp-open-lwip/include
+	ifeq ($(LWIP_FLAVOUR), lwip2)
+		THIRD_PARTY_DATA += third-party/lwip2/Makefile.sming
+		EXTRA_INCDIR += compiler/include/lwip2
```

Comment:
@d-a-v The "compiler" directory is not the right place to put the include files for LWIP 2. They should be part of the submodule and ideally should end up in `third-party/lwip2/include`